r-package/grf/DESCRIPTION

```diff
@@ -32,5 +32,8 @@ RoxygenNote: 6.1.1
 Suggests:
     DiagrammeR,
     testthat,
+    sufrep
+Remotes:
```

Comment:
R CMD check complains, and in this thread https://github.com/r-lib/devtools/issues/1717#issuecomment-368125654 it's mentioned that Remotes should not be there